ReactWindows/ReactNative/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,326 @@
+using ReactNative.Bridge;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace ReactNative.Common
+{
+    /// <summary>
+    /// Accessibility helper methods.
+    /// </summary>
+    public static class AccessibilityHelper
+    {
+        #region ImportantForAccessibility - Provides support for ImportantForAccessibility property implementation.
+
+        /// <summary>
+        /// Initializes the value of AccessibilityView for all the children of <paramref name="parentUIElement"/>,
+        /// based on the <paramref name="parentUIElement"/> and its ancesstor settings. Must be called
```

Comment:
>ancesstor [](start = 68, length = 9)

typo

Reply:
Fixed.

---
In reply to: [166811286](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r166811286) [](ancestors = 166811286)